src/lib/scopes.ts

```diff
@@ -60,12 +60,14 @@ export function convertScopesToExternal (
     const accountId: string = scope.accountId
 
     if (!(accountId in scopeDictionary)) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
let not silence complaints, better is to drop @ts-ignore and apply properly types

Reply:
existing linting issues will be fixed in separate story.